src/components/OpenStreetMap.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies.
+ */
+import { sprintf, __ } from '@wordpress/i18n';
+import { useEffect } from '@wordpress/element';
+
+/**
+ * Internal Dependencies.
+ */
+import { getFromGlobal } from '../helpers/globals';
+
+/**
+ * Leaflet.
+ */
+import L from 'leaflet';
```

Comment:
`import Leaflet from 'leaflet';` so it's more readable. Thx!

Reply:
@mauteri Done